grayskull/pypi/pypi.py

```diff
@@ -911,28 +945,30 @@ def _get_run_req_from_requires_dist(self, requires_dist: List) -> List:
         :return:
         """
         run_req = []
-        for req in requires_dist:
-            list_raw_requirements = req.split(";")
-            selector = ""
-            if len(list_raw_requirements) > 1:
-                list_extra = PyPi._get_extra_from_requires_dist(
-                    list_raw_requirements[1]
+        print(f"This is requires_dist: {requires_dist}")
+        if requires_dist:
+            for req in requires_dist:
+                list_raw_requirements = req.split(";")
+                selector = ""
+                if len(list_raw_requirements) > 1:
+                    list_extra = PyPi._get_extra_from_requires_dist(
+                        list_raw_requirements[1]
+                    )
+                    if PyPi.__skip_pypi_requirement(list_extra):
+                        continue
+
+                    result_selector = self._get_all_selectors_pypi(list_extra)
+
+                    if result_selector:
+                        selector = " ".join(result_selector)
+                        selector = f"  # [{selector}]"
+                    else:
+                        selector = ""
+                pkg_name, version = PyPi._get_name_version_from_requires_dist(
+                    list_raw_requirements[0]
                 )
-                if PyPi.__skip_pypi_requirement(list_extra):
-                    continue
-
-                result_selector = self._get_all_selectors_pypi(list_extra)
-
-                if result_selector:
-                    selector = " ".join(result_selector)
-                    selector = f"  # [{selector}]"
-                else:
-                    selector = ""
-            pkg_name, version = PyPi._get_name_version_from_requires_dist(
-                list_raw_requirements[0]
-            )
-            run_req.append(f"{pkg_name} {version}{selector}".strip())
-        return run_req
+                run_req.append(f"{pkg_name} {version}{selector}".strip())
+            return run_req
```

Comment:
This has got indented a bit too much I think. It might be not reachable under certain code paths.